lib/climate_control/modifier.rb

```diff
@@ -33,10 +33,24 @@ def run_block
       @block.call
     end
 
+    def stringify_boolean(value)
+      return value.to_s if [true, false].include?(value)
+      value
+    end
+
+    def stringify_number(value)
+      return value.to_s if value.is_a?(Numeric)
+      value
+    end
+
+    def stringify_value(value)
+      stringify_number(stringify_boolean(value))
+    end
+
     def copy_overrides_to_environment
       @environment_overrides.each do |key, value|
         begin
-          @env[key] = value
+          @env[key] = stringify_value(value)
```

Comment:
Thoughts on doing `value.to_s` here? This should accomplish the same thing without any of the `stringify_` methods you defined. It also allows you to pass in whatever you want as long as you define a `to_s` method.